tests/slepos/build_images_creator.pm

```diff
@@ -0,0 +1,256 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Build SLEPOS images using image creator
+# Maintainer: Pavel Sladek <psladek@suse.cz>
+
+use base "basetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use slepos_images;
+
+
+my %template_done;
+
+sub build_image {
+
+    my ($image, $template, $template_idx, $packages_ar) = @_;
+
+    my ($imagebase, $version) = split_image_name($image);
+    bmwqemu::diag("image '$image' will be built using image creator:");
```

Comment:
please avoid use of `bmwqemu`. `diag` is part of testapi directly since some time. The ":" does not make sense to me as there is nothing coming afterwards